<?xml version="1.0"?>
<launch>  
  
    <node name="controller" pkg="pid" type="controller" ns="yaw_angle" output="screen" >
      <param name="node_name" value="yaw_angle_pid" />
      <param name="Kp" value="0.32" />
      <param name="Ki" value="0.01" />
      <param name="Kd" value="0.0" />
      <param name="upper_limit" value="10" />
      <param name="lower_limit" value="-10" />
      <param name="windup_limit" value="10" />
      <param name="max_loop_frequency" value="100.0" />
      <param name="min_loop_frequency" value="100.0" />
      <remap from="setpoint" to="/setpoint_yaw" />
     </node>

    <node name="controller" pkg="pid" type="controller" ns="dist" output="screen" >
      <param name="node_name" value="dist_pid" />
      <param name="Kp" value="3.2" />
      <param name="Ki" value="0.56" />
      <param name="Kd" value="0.04" />
      <param name="upper_limit" value="10" />
      <param name="lower_limit" value="-10" />
      <param name="windup_limit" value="10" />
      <param name="max_loop_frequency" value="100.0" />
      <param name="min_loop_frequency" value="100.0" />
      <remap from="setpoint" to="/setpoint_dist" />
     </node>

</launch>